dll/ntdll/rtl/version.c

```diff
@@ -89,12 +90,29 @@ SetRosSpecificInfo(IN OUT PRTL_OSVERSIONINFOEXW VersionInformation)
  *
  * REVISIONS
  *  2000-08-10 ekohl
+ *  2020-03-03 Fraizeraust
  *
  * @implemented
  */
 BOOLEAN NTAPI
 RtlGetNtProductType(_Out_ PNT_PRODUCT_TYPE ProductType)
 {
+    /* Raise an exception if the passed argument is NULL */
+    if (ProductType == NULL)
+    {
+        RtlRaiseStatus(STATUS_ACCESS_VIOLATION);
```

Comment:
Why?
Just writing to the pointer will cause this exception?
Seems weird to explicitly raise it (and in fact, makes it harder to debug since a 'real' access violation includes info about the arguments)